lib/ts/index.ts

```diff
@@ -54,6 +54,22 @@ export default class SuperTokensAPIWrapper {
         return SuperTokens.getSuperTokensRoutesForReactRouterDom(reactRouterDom);
     }
 
+    static redirectToAuthWithRedirectToPath(
+        show?: "signin" | "signup",
+        history?: any,
+        queryParams?: any
+    ): Promise<void> {
+        return SuperTokens.getInstanceOrThrow().redirectToAuthWithRedirectToPath(show, history, queryParams);
+    }
+
+    static redirectToAuthWithoutRedirectToPath(
+        show?: "signin" | "signup",
+        history?: any,
+        queryParams?: any
+    ): Promise<void> {
+        return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(show, history, queryParams);
+    }
```

Comment:
We should combine into one function (`redirectToAuth`) and have an option called `redirectBack` (just like we have now):
```ts
static redirectToAuth(options: {
        show?: "signin" | "signup",
        history?: any,
        queryParams?: any
        redirectBack?: boolean
    }): Promise<void> {
        return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(show, history, queryParams);
    }
```